### PR TITLE
RavenDB-9686: Minor refactoring

### PR DESCRIPTION
**Pull request by aviviadi**

### CODE REVIEW

File: src/Raven.Server/Config/Categories/DatabaseConfiguration.cs [hunk 1]
```diff
@@ -13,29 +13,29 @@ public class DatabaseConfiguration : ConfigurationCategory
         public TimeSetting OperationTimeout { get; set; }
 
         /// <summary>
-        /// This much time has to wait for the resource to become available when too much
-        /// different resources get loaded at the same time
+        /// This much time has to wait for the database to become available when too much
+        /// different datavases get loaded at the same time
```
Comment (ml054) [inline src/Raven.Server/Config/Categories/DatabaseConfiguration.cs:17]: typo

File: src/Raven.Server/Config/Categories/DatabaseConfiguration.cs [hunk 2]
```diff
@@ -13,29 +13,29 @@ public class DatabaseConfiguration : ConfigurationCategory
...
         /// </summary>
         [Description("The time in seconds to wait for a database to start loading when under load")]
         [DefaultValue(10)]
         [TimeUnit(TimeUnit.Seconds)]
-        [ConfigurationEntry("Databases.ConcurrentResourceLoadTimeoutInSec", ConfigurationEntryScope.ServerWideOnly)]
-        public TimeSetting ConcurrentResourceLoadTimeout { get; set; }
+        [ConfigurationEntry("Databases.ConcurrentDatabaseLoadTimeoutInSec", ConfigurationEntryScope.ServerWideOnly)]
```
Comment (ppekrol) [inline src/Raven.Server/Config/Categories/DatabaseConfiguration.cs:22]: We are already in `Databases.` so maybe ConcurrentLoadTimeoutInSec would be enough?

File: src/Raven.Server/Config/Categories/DatabaseConfiguration.cs [hunk 3]
```diff
@@ -13,29 +13,29 @@ public class DatabaseConfiguration : ConfigurationCategory
...
+        public TimeSetting ConcurrentDatabaseLoadTimeout { get; set; }
 
         /// <summary>
         /// specifies the maximum amount of databases that can be loaded simultaneously
         /// </summary>
         [DefaultValue(8)]
-        [ConfigurationEntry("Databases.MaxConcurrentResourceLoads", ConfigurationEntryScope.ServerWideOnly)]
-        public int MaxConcurrentResourceLoads { get; set; }
+        [ConfigurationEntry("Databases.MaxConcurrentDatabaseLoads", ConfigurationEntryScope.ServerWideOnly)]
```
Comment (ppekrol) [inline src/Raven.Server/Config/Categories/DatabaseConfiguration.cs:29]: Maybe change to MaxConcurrentLoads?

File: src/Raven.Server/Config/Categories/DatabaseConfiguration.cs [hunk 4]
```diff
@@ -13,29 +13,29 @@ public class DatabaseConfiguration : ConfigurationCategory
...
+        public int MaxConcurrentDatabaseLoads { get; set; }
 
         [DefaultValue(900)]
         [TimeUnit(TimeUnit.Seconds)]
-        [ConfigurationEntry("Databases.MaxIdleTimeForTenantDatabaseInSec", ConfigurationEntryScope.ServerWideOnly)]
-        [LegacyConfigurationEntry("Raven/Databases/MaxIdleTimeForTenantDatabase")]
+        [ConfigurationEntry("Databases.MaxIdleTimeForDatabaseInSec", ConfigurationEntryScope.ServerWideOnly)]
+        [LegacyConfigurationEntry("Raven/Databases/MaxIdleTimeForDatabase")]
```
Comment (ppekrol) [inline src/Raven.Server/Config/Categories/DatabaseConfiguration.cs:35]: We should not change LegacyConfigurationEntry value

File: src/Raven.Server/Config/Categories/DatabaseConfiguration.cs [hunk 5]
```diff
@@ -13,29 +13,29 @@ public class DatabaseConfiguration : ConfigurationCategory
...
-        [ConfigurationEntry("Databases.MaxConcurrentResourceLoads", ConfigurationEntryScope.ServerWideOnly)]
-        public int MaxConcurrentResourceLoads { get; set; }
+        [ConfigurationEntry("Databases.MaxConcurrentDatabaseLoads", ConfigurationEntryScope.ServerWideOnly)]
+        public int MaxConcurrentDatabaseLoads { get; set; }
 
         [DefaultValue(900)]
         [TimeUnit(TimeUnit.Seconds)]
-        [ConfigurationEntry("Databases.MaxIdleTimeForTenantDatabaseInSec", ConfigurationEntryScope.ServerWideOnly)]
-        [LegacyConfigurationEntry("Raven/Databases/MaxIdleTimeForTenantDatabase")]
+        [ConfigurationEntry("Databases.MaxIdleTimeForDatabaseInSec", ConfigurationEntryScope.ServerWideOnly)]
```
Comment (ppekrol) [inline src/Raven.Server/Config/Categories/DatabaseConfiguration.cs:34]: Maybe change to MaxIdleTimeInSec?

**Pull request merged**